web/src/main/webapp/xsl/inspire-atom-feed.xsl

```diff
@@ -0,0 +1,348 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
webapp/xsl folder will be remove at some point. You should now use webapp/xslt folder and in this case probably in services/inspire-atom ?